R/latex.R

```diff
@@ -238,7 +238,7 @@ latexmk_emu = function(
   }
   for (i in seq_len(times)) {
     if (file.exists(logfile)) {
-      if (!any(grepl('(Rerun to get|Please \\(re\\)run) ', readLines(logfile), useBytes = TRUE))) break
+      if (!any(grepl('(Rerun to get|Please \\(re\\)run) | Rerun LaTeX\\.', readLines(logfile), useBytes = TRUE))) break
```

Comment:
` | Rerun LaTeX\\.` needs to be moved into `()`:

```suggestion
      if (!any(grepl('(Rerun to get|Please \\(re\\)run | Rerun LaTeX\\.)', readLines(logfile), useBytes = TRUE))) break
```

Reply:
In that case, should I add an extra space after "get"?  That would preserve the original regex.

Reply:
Oh yes.